client/app/styles/rjsf/InlineFieldTemplate.tsx

```diff
@@ -37,7 +37,10 @@ function InlineFieldTemplate({
   const error = rawErrors && rawErrors[0];
 
   // UI Schema options
-  const isLabel = uiSchema?.["ui:options"]?.label !== false;
+  const options = uiSchema?.["ui:options"] || {};
+  const jsxTitle = options?.jsxTitle as any;
+  const fieldLabel = jsxTitle ?? label;
+  const isLabel = options?.label !== false && fieldLabel;
```

Comment:
Well the `FileWidget` here is wrapped in the `InlineFieldTemplate` so I think there is a misunderstanding and it seems odd to hardcode this into the `FileWidget`.

This is definitely a nice feature as it allows us to add bespoke titles to our fields if the designs require them like in this case. We are using it already in the `TitleOnlyFieldTemplate` and realistically it would be good to add in all field templates as required.